app/models/product.rb

```diff
@@ -0,0 +1,7 @@
+class Product < ApplicationRecord
+    has_one_attached :avatar
+
+    scope :filtered, ->(filter_param) {where(:description => filter_param)}
+    scope :vegetarian, ->() {where(:vegetarian => "true")}
+    scope :filter_price, ->(slide1,slide2) {where("price>? AND price<?",slide1,slide2)}
```

Comment:
Please follow coding style best practices `scope :filter_price, -> (slide1, slide2) { where('price > ? AND price < ?', slide1, slide2) }`.
Also use double quotes only when you need to interpolate strings.